cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,71 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require("../support/generate");
+
```

Comment:
```suggestion
	const book = {
		title: ...
		description: ...
	}
```
and then reuse in tests like `book.title`